test/binaries/test-gnutls-serv-2.1.6.c

```diff
@@ -5,7 +5,7 @@ int main() {
 	printf("It outputs a few strings normally associated with gnutls-serv 2.1.6");
 	printf("They appear below this line.");
 	printf("------------------");
-	printf("gnutls-serv 2.1.6");
+	printf("gnutls-cli 2.1.6");
```

Comment:
Was this a mistake or does the filename need to be changed to also be gnutls-cli?

Reply:
No previous version had it like that.

Reply:
Maybe both binary has same version signature.

Reply:
I think I'd added this testcase when something wasn't working at one point...

Reply:
Hmmm, no wasn't me. Ya I'm of the mind these probably need to be changed back, or we need to be sure of why they're chaging

Reply:
Ah I found what I was thinking of though: ff14b13797b4aaf3911adde3ba40e0e14b1d810b

Maybe I did something up with that change?

Or the next possibly relevant change after that is dbca3aaa776166b9c7c5147a3c99bb406f8f4424

> `git log -p -- cve_bin_tool/checkers/gnutls.py`

Reply:
That's weird.  Let's just fix it now, then?  or open an issue to fix it after if you want to get this PR merged ASAP?